pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -593,6 +598,14 @@ def get_mount_point_constraints(self):
                 constraint = self._get_mount_point_constraints_data(p)
                 if p.mountpoint == "/boot":
                     constraint.recommended = True
+                # On MBR neither a BIOS boot partition (typically labeled as biosboot) nor a /boot/efi partition is required.
+                if p.fstype == "biosboot" or p.mountpoint == "/boot/efi":
+                    stage1_disk = self.storage.bootloader.stage1_disk
+                    stage1_device = stage1_disk and self.storage.devicetree.resolve_device(stage1_disk)
+                    is_gpt = (stage1_device and
+                              getattr(stage1_device.format, "label_type", None) == "gpt")
+                    if is_gpt:
+                        constraint.required = True
                 else:
```

Comment:
We need to fix the logic a bit.
Here all the non-biosboot or /boot/efi constraints will be set as required